Cargo.toml

```diff
@@ -195,6 +195,8 @@ tracing = { workspace = true }
 which = { workspace = true }
 uuid = { workspace = true }
 url = { workspace = true, features = ["serde"] }
+tokio = { workspace = true, features = ["rt", "rt-multi-thread", "macros"] }
```

Comment:
Can't highlight the line, but, seems like maybe we can get rid of all the `reqwest` "blocking" features now?

Reply:
good call out! removing